src/scripts/main.js

```diff
@@ -1,5 +1,47 @@
 'use strict';
 
+const spider = document.querySelector('.spider');
+const theWall = document.querySelector('.wall');
+const theWallWidth = parseInt(getComputedStyle(theWall).width);
+const theWallHeight = parseInt(getComputedStyle(theWall).height);
+const theWallBorder = parseInt(getComputedStyle(theWall).border);
```

Comment:
why just don't use clientWidth and clientHeight?
https://learn.javascript.ru/size-and-scroll